src/components/JokeCard.jsx

```diff
@@ -0,0 +1,22 @@
+
+import { Box, Text } from "@chakra-ui/react";
+import { RiDoubleQuotesL } from 'react-icons/ri';
+
+export function JokeCard({ joke }) {
+  return(
+    <Box 
+      bg='gray.100'
+      borderRadius="5"
+      minWidth={["17rem", "23rem", "26rem"]}
+      maxWidth={["95%", "45%", "30%"]}
+      padding="8"
+      paddingLeft="16"
```

Comment:
If you want to use chakra spacing system, it's better to pass it as a number instead of string, so for example `padding={8}`. Check all places where you are using spacing.